src/test/data/JsonSerializableStudentDirectoryTest/typicalStudentsStudentDirectory.json

```diff
@@ -12,41 +12,47 @@
     "address" : "311, Clementi Ave 2, #02-25",
```

Comment:
The other json files with students should be updated as well

Reply:
Additionally, some students should have makeup lessons, and students without makeup lessons should have the field be `[ ]` (instead of `null`)